src/features/campaigns/components/ActivitiesOverview/ActivitiesOverviewCard.tsx

```diff
@@ -92,14 +101,21 @@ const ActivitiesOverviewCard: FC<OverviewListProps> = ({
         }
       })}
       {numExtra > 0 && (
-        <Box textAlign="center">
-          <Typography>
-            <Msg
-              id={messageIds.activitiesOverview.extraActivities}
-              values={{ numExtra }}
-            />
-          </Typography>
-        </Box>
+        <NextLink
+          href={`/organize/${orgId}/projects${
+            campId ? `/${campId}` : ''
+          }/calendar?focusDate=${today}&timeScale=${timeScale}`}
```

Comment:
Why use `today` as the focus date here, and not `focusDate` that is already available as a property in this component?